### PR TITLE
    feat(dashboard): manage static page

### PR DESCRIPTION
**Pull request by EricDav**
#### What does this PR do?
Implement static page management
#### Description of Task to be completed?
Create static page
Edit static page
Delete static page
#### How should this be manually tested?
npm run test-local
#### Any background context you want to provide?
Nill
#### What are the relevant pivotal tracker stories?
#150749931
#### Screenshots (if appropriate)
screen shot 2017-09-20 at 7 47 07 pm
screen shot 2017-09-20 at 7 52 54 pm

#### Questions:
Nill

### CODE REVIEW

File: imports/plugins/core/ui-navbar/client/components/navbar/navbar.js [hunk 1]
```diff
@@ -73,6 +73,14 @@ Template.CoreNavigationBar.helpers({
       }
     };
   },
+
+  staticPagesMenu() {
```
Comment (temilaj) [inline imports/plugins/core/ui-navbar/client/components/navbar/navbar.js:77]: I'm curious to know why this function is documented.

Comment (EricDav) [reply imports/plugins/core/ui-navbar/client/components/navbar/navbar.js:77]: I did not get you. Do you mean why it is not documented?

Comment (EricDav) [reply imports/plugins/core/ui-navbar/client/components/navbar/navbar.js:77]: I did not get you. do you mean why it is not documented?

Comment (temilaj) [reply imports/plugins/core/ui-navbar/client/components/navbar/navbar.js:77]: Yes, I do.

**Pull request closed without merging**